packages/mobile/src/features/videos/components/MainScreen/SmallHeaderButton.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FontWeight } from '../../../common/styles/constans/FontWeight';
+import { FontFamily } from '../../../common/styles/constans/FontFamily';
+import { FontSize } from '../../../common/styles/constans/FontSize';
+import { Color } from '../../../common/styles/constans/Color';
+import { MyText } from '../../../common/styles/MyText';
+
+const Wrapper = styled.TouchableOpacity`
+`;
+
+const ButtonFilter = styled.TouchableOpacity({
+    //marginRight: 10,
+    height: 26,
+    minWidth: 51,
+    //flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderRadius: 10,
+    borderColor: Color.LIGHTGREYXX,
+    //border: 1,  ??
```

Comment:
jak zakomentowałeś i nie są Ci potrzebne to usuń

Reply:
ok